apps/modernization-api/src/main/java/gov/cdc/nbs/service/util/Constants.java

```diff
@@ -8,6 +8,13 @@ private Constants() {
 
     //Application ID
     public static final String APP_ID = "PATIENT-SEARCH";
+    
+    //Patient Update Types
+    public static final String UPDATE_GENERAL_INFO ="GeneralInformation";
+    
+    public static final String UPDATE_SEX_BIRTH="Sex&Birth";
+    
+    public static final String UPDATE_MORTALITY="Mortality";
```

Comment:
Instead of duplicating these constants here and in the patient-listener, we could move them into the `event-schema` library since they pertain to the kafka messaging.

Reply:
@mpeels updated